sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/OpenTelemetryHttpPolicy.java

```diff
@@ -48,7 +48,19 @@ public HttpPipelinePolicy create() {
     }
 
     // Singleton OpenTelemetry tracer capable of starting and exporting spans.
-    private static final Tracer TRACER = GlobalOpenTelemetry.getTracer("Azure-OpenTelemetry");
+    private final Tracer tracer;
+
+    /**
+     * Creates new OpenTelemetry {@code HttpPipelinePolicy} with default global tracer
+     * {@code GlobalOpenTelemetry.getTracer}
+     */
+    public OpenTelemetryHttpPolicy() {
+        this(GlobalOpenTelemetry.getTracer("Azure-OpenTelemetry"));
+    }
+
+    OpenTelemetryHttpPolicy(Tracer tracer) {
```

Comment:
This is for testing purposes only, right? If so, mind adding a small comment indicating that.